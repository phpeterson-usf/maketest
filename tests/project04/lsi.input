-h $(repo_dir)/lsi.hex $(TESTS_DIR)/lsi.s
