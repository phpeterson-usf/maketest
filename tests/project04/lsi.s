main:
    lsl r2, r3, #2
    bx lr
