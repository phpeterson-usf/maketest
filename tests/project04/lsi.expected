v2.0 raw
e1a02103
e12fff1e
